libfwupdplugin/fu-udev-device.c

```diff
@@ -1241,6 +1241,28 @@ fu_udev_device_write_sysfs (FuUdevDevice *self, const gchar *attribute,
 #endif
 }
 
+/**
+ * fu_udev_device_get_devtype
+ * @self: A #FuUdevDevice
+ *
+ * Returns the Udev device type
+ *
+ * Returns: device type specified in the uevent
+ *
+ * Since: 1.5.0
```

Comment:
Do we want this to go into 1.4.5 eventually too?

Reply:
Maybe. I wasn't sure since I couldn't test it.

Reply:
i think this kind of safe/obvious stuff can just go into 1.4.5 even without the retimer bits.

Reply:
OK, adjusted it.